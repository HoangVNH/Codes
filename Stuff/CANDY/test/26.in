10
43
XXXXXXXXDDDDDDDDDDDXXVVVVVXXXXXXDDDVVVVXXXV
26
VVVXXXXXDDDDDXXXXXXXXDXXXX
26
XXXXXDDDXXXXVVVVDXXDDDDXXX
49
DDDDVVVDDDDDDDVDDDXXXXXXXXVVVVVDDDDDXXXDXXXDDDXVV
37
XXXDDDDDXDXXXXVVVVVVVVVXXXXXVVVVDDDXX
29
VXXXXVXXXDDDDXXXXXXXXXXXXXXXD
46
DDVVXXXXXXXXXXXXXXXXXXXDDDDDDDDVVVDXXVVVVVDDDD
21
XXXXXXVVVVVXXXVVVVDXX
50
VVVXXXXXVVVVXXXXXDDDDDDDDDVVVVXXVVVVVVXXXDDDDDDDDD
42
DDDDDXXVXXXXXXXDVVDVVVVDDDDXXXXXXXVVVXVVVV
