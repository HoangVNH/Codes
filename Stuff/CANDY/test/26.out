42
25
24
47
35
28
45
19
49
41
